OpenSim/Actuators/Test/testActuators.cpp

```diff
@@ -322,7 +322,7 @@ void testTorqueActuator()
 
     model->computeStateVariableDerivatives(state);
 
-    double fKE = model->getMatterSubsystem().calcKineticEnergy(state);
+    /*double fKE = */model->getMatterSubsystem().calcKineticEnergy(state);
```

Comment:
The difference in final and initial KE should be used to compare to the actuator work. This is a major part of the test.